.github/workflows/build-nym-vpn-x-linux.yml

```diff
@@ -0,0 +1,124 @@
+name: build-nym-vpn-x-linux
+on:
+  workflow_dispatch:
+  workflow_call:
+    secrets:
+      TAURI_PRIVATE_KEY:
+        required: true
+      TAURI_KEY_PASSWORD:
+        required: true
+      DESKTOP_JS_SENTRY_DSN:
+        required: true
+    outputs:
+      UPLOAD_DIR_LINUX:
+        value: ${{ jobs.build-linux.outputs.UPLOAD_DIR_LINUX }}
+      PKG_VERSION:
+        value: ${{ jobs.build-linux.outputs.PKG_VERSION }}
+      RUST_VERSION:
+        value: ${{ jobs.build-linux.outputs.RUST_VERSION }}
+
+env:
+  CARGO_TERM_COLOR: always
+  CARGO_TARGET: release
+  UPDATER_BUNDLE_NAME: updater_linux_x86_64.AppImage.tar.gz
+  UPLOAD_DIR_LINUX: linux_artifacts
+
+jobs:
+  build-linux:
+    runs-on: ubuntu-22.04-arc
+    outputs:
+      UPLOAD_DIR_LINUX: ${{ env.UPLOAD_DIR_LINUX }}
+      PKG_VERSION: ${{ steps.package-version.outputs.metadata }}
+      RUST_VERSION: ${{ steps.rust-version.outputs.rustc }}
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v4
+
+      - name: Install system dependencies
+        run: |
+          sudo apt-get update && sudo apt-get install -y libdbus-1-dev libmnl-dev libnftnl-dev \
+            libwebkit2gtk-4.0-dev build-essential curl wget libssl-dev jq \
+            libgtk-3-dev squashfs-tools libayatana-appindicator3-dev make libfuse2 unzip librsvg2-dev file
+
+      - name: Install rust toolchain
+        uses: brndnmtthws/rust-action-rustup@v1
+        with:
+          toolchain: stable
+          components: rustfmt, clippy
+
+      - name: Install cargo-about
+        run: cargo install --locked cargo-about
+
+      - name: Install Node
+        uses: actions/setup-node@v4
+        with:
+          node-version: 20
+
+      - name: Install Protoc
+        uses: arduino/setup-protoc@v3
```

Comment:
Surprised to see you finally switched to latest stable protoc (which is a good thing). What changed since you were blocking to keep with the outdated ubuntu system protoc? (https://github.com/nymtech/nym-vpn-client/pull/565)